### PR TITLE
Add unit test for `_fixImpliedMeridiemOfChronoResult `

### PR DESCRIPTION
**Pull request by arthurshir**
- Test implied meridian text adjustment output
- Test implied meridian date output
- Add export for `_fixImpliedMeridiemOfChronoResult` function
- Update the TIME_ZONE constant to be consistent with naming convention of other files

### CODE REVIEW

File: tst/nlp/parseNonRecurringSingleDate.test.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
 const parseNonSingleRecurringDate = require('../../nlp/parseNonRecurringSingleDate.js'),
+    moment = require('moment-timezone'),
     timemachine = require("../../timemachine.js");
 
+const TIME_ZONE = "America/Los_Angeles";
```
Comment (arthurshir) [inline tst/nlp/parseNonRecurringSingleDate.test.js]: Use TIMEZONE

File: tst/nlp/parseNonRecurringSingleDate.test.js [hunk 2]
```diff
@@ -48,4 +51,62 @@ describe("_convertOnTimetoAtTime", () => {
             expect(parseNonSingleRecurringDate._convertOnTimetoAtTime(key)).toEqual(map[key]);
         }
     });
+});
```
Comment (arthurshir) [inline tst/nlp/parseNonRecurringSingleDate.test.js]: bubakazouba 4 minutes ago  Owner
lets add a couple of tests where the function takes the full date/time and not just the time
and another edge case for when the function takes a date without at time

**Comment by arthurshir** [comment]
remove what you've done from the TODOS.java

File: tst/nlp/parseNonRecurringSingleDate.test.js [hunk 3]
```diff
@@ -48,4 +51,62 @@ describe("_convertOnTimetoAtTime", () => {
...
+            'at 5:59': {
+                result__text: 'at 5:59 pm',
+                d: moment.parseZone("2018-06-03T17:59:00.000-07:00")
+            },
+            'at 6:00': {
+                result__text: 'at 6:00 pm',
+                d: moment.parseZone("2018-06-03T18:00:00.000-07:00")
+            },
+
+            // Text shouldn't change
```
Comment (bubakazouba) [inline tst/nlp/parseNonRecurringSingleDate.test.js]: Let's remove these test cases, After analyzing the code, it looks like the following happens:
1- function takes "at 6:00 pm"
2- appends " pm" and " am" to it, so the following gets passed to chrono:
    2.1- "at 6:00 pm pm"
    2.2- "at 6:00 pm am"
3- chrono simply ignores the other meridiem and just picks the first "pm"

We shouldnt test this behaviour as it's not guaranteed that it would be consistent. If chrono decides tomorrow to pick the second meridiem the unit test shouldn't break.

Note: The function should only be called to fix the implied Meridiem, If the Meridiem is not implied then it shouldn't be called.

------

If you still think it's a good idea to unit test these cases, modify the function to handle non implied meridiems. So it would first check if the meridiem is not implied, and if that's the case it would throw an exception, and then you can assert on the exception.
